pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -262,7 +262,9 @@ def circuit(num_qubits):
     # pylint: disable=too-many-instance-attributes
 
     # So far we just consider the options for MPS simulator
-    _device_options = ("backend", "max_bond_dim", "cutoff", "cutoff_mode")
+    _device_options_mps = ("backend", "max_bond_dim", "cutoff", "cutoff_mode")
+    _device_options_tn = ("backend", "cutoff", "cutoff_mode")
```

Comment:
What do you think about having a small dict here? `_device_options_mps=_device_options["mps"]`.
This way you could call the device options as _device_options[self._method]. 
Also, we can iterate over it in the future if needed.

Reply:
Good idea. I will do it